src/tools/gophish_complete.py

```diff
@@ -1,9 +1,6 @@
 #!/usr/bin/env python3
 
-"""GoPhish Campaign Completion tool for a Phishing Campaign Assessment (PCA).
-
-User interface to select a campaign to complete or view a summery.  If auto
-flag is provided with a CAMPAIGN_ID that campaign will be completed.
+"""Complete a campaign and/or output a campaign summary.
```

Comment:
Similar to previous comment, I'd say something like `"Compete a campaign in GoPhish and/or output a GoPhish campaign summary."`.